lib/gollum/public/gollum/javascript/editor/gollum.editor.js

```diff
@@ -51,10 +51,37 @@
       }).insertAfter(textarea);
       textarea.css('display', 'none');
 
+      // Create a div for the autosave status message
+      var autoSaveMessage = document.createElement('div');
+      $(autoSaveMessage).addClass('div position-fixed bottom-0 left-0');
+      $('body').append(autoSaveMessage);
+
+      var autoSaveTimer = null;
+
       // NOTE: This requires the page to have only one 'gollum-editor-body'.
```

Comment:
There is only one gollum-editor-body, but there may be up to three more text fields for the `_Footer`, `_Header` and `_Sidebar`. How difficult would it be to add the content from those text fields to the local storage as well?

Reply:
Probably not too difficult. I'll look into adding those too.